### PR TITLE
Move/update SECURITY.md to root, add to staging repos

### PR DESCRIPTION
**Pull request by joelsmith**
See https://github.com/kubernetes/security/issues/105

```release-note
NONE
```

### CODE REVIEW

**Comment by nikhita** [comment]
/assign @liggitt 
for approval

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Security Policy
```
Comment (nikhita) [inline SECURITY.md:1]: @joelsmith looks like we already have a SECURITY.md for k/k at https://github.com/kubernetes/kubernetes/blob/master/.github/SECURITY.md

Comment (nikhita) [reply SECURITY.md:1]: Also shows up here - https://github.com/kubernetes/kubernetes/security/policy

**Comment by joelsmith** [comment]
I wasn't aware that the `SECURITY.md` file could also live in `.github/`.  So I suppose that the choice comes down to whether to put it in the root for increased visibility or to put it in `.github` to avoid adding things to the root dir. I see that in #79050 they decided to avoid clutter. I think that my preference is for increased visibility. I suspect that most GitHub users are unaware that that there is a repo "Security" tab at the top of the website. I had no idea to look there until we started talking about adding the policies across all the repos.

/cc @tallclair

**Comment by harche** [comment]
> @joelsmith: The following test **failed**, say `/retest` to rerun all failed tests:
> 
> Test name	Commit	Details	Rerun command
> pull-kubernetes-node-crio1-18-e2e	[602a517](https://github.com/kubernetes/kubernetes/commit/602a517b794edcc79a623041af849f0ba2a22827)	[link](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/93486/pull-kubernetes-node-crio1-18-e2e/1288353857021153280/)	`/test pull-kubernetes-node-crio1-18-e2e`
> [Full PR test history](https://prow.k8s.io/pr-history?org=kubernetes&repo=kubernetes&pr=93486). [Your PR dashboard](https://prow.k8s.io/pr?query=is%3Apr%20state%3Aopen%20author%3joelsmith). Please help us cut down on flakes by [linking to](https://git.k8s.io/community/contributors/devel/sig-testing/flaky-tests.md#filing-issues-for-flaky-tests) an [open issue](https://github.com/kubernetes/kubernetes/issues?q=is:issue+is:open) when you hit one in your PR.

@joelsmith Please ignore this test error, I am testing manual trigger of one crio test using this PR. Thanks.

**Comment by joelsmith** [comment]
I have updated the PR to update `/.github/SECURITY.md` and move it to `/SECURITY.md`.

**Comment by nikhita** [comment]
@joelsmith would this still be necessary if we have org-wide defaults - https://github.com/kubernetes/org/issues/2081?

**Comment by joelsmith** [comment]
@nikhita at a minimum I would prefer to have the file in the root of k/k for visibility. I don't think that the GH security policy link is well known yet, so I'm afraid people won't know to look for it.  We get enough people improperly reporting security issues as it is. I don't feel very strongly either way for the staging repos.  Perhaps @tallclair could chime in too.

**Comment by nikhita** [comment]
@joelsmith Yeah, I don't have strong opinions on this. Just wanted to confirm :smile:

**Comment by liggitt** [comment]
/skip

sorry for the radio silence, the notifications for this got dropped. is this ready/still desired?

**Comment by nikhita** [comment]
> sorry for the radio silence, the notifications for this got dropped. is this ready/still desired?

Let's hold this until we decide how to structure security contacts - https://github.com/kubernetes/community/pull/5398
Ref: sig-arch thread - https://groups.google.com/g/kubernetes-sig-architecture/c/0MUJBDGf3jg/m/wYrr8i1tBQAJ

/hold

**Comment by liggitt** [comment]
ack, feel free to reassign me for approval once needed

**Pull request closed without merging**